src/Controller/HomeController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
```

Comment:
Please remove this doc block from the beginning of each PHP file. Probably it was added by PhpStorm so make sure it is disabled. I guess in the final version we will have a different text as licence here.